src/modules/stats/get/dto.ts

```diff
@@ -2,15 +2,22 @@ import Validation from '../../../tools/validation';
 import type { IGetStatsDto } from './types';
 
 export default class UserStatsDto implements IGetStatsDto {
-  id: string;
+  id?: string;
+  character?: string;
+  lvl?: number;
 
   constructor(data: IGetStatsDto) {
     this.id = data.id;
-
+    this.character = data.character;
+    this.lvl = data.lvl;
     this.validate();
   }
 
   private validate(): void {
-    new Validation(this.id, 'id').isDefined().isString().isObjectId();
```

Comment:
This should still throw "No id provided" when user provides no data. This way, user will return empty data instead of informing, whats missing.